llvm/include/llvm/BinaryFormat/ELFRelocs/AArch64.def

```diff
@@ -166,6 +167,7 @@ ELF_RELOC(R_AARCH64_P32_ADR_GOT_PAGE,                0x01a)
 ELF_RELOC(R_AARCH64_P32_LD32_GOT_LO12_NC,            0x01b)
 ELF_RELOC(R_AARCH64_P32_LD32_GOTPAGE_LO14,           0x01c)
 ELF_RELOC(R_AARCH64_P32_PLT32,                       0x01d)
+ELF_RELOC(R_AARCH64_P32_GOTPCREL32,                  0x01e)
```

Comment:
Can you take this out? One comment I got internally in the ABI was that it was not worth defining an ILP32 relocation for this as the number of available codes is much more limited than ILP64 (ELF32 vs ELF64).

I think we can define this if there is a need to deploy this on an ILP32 platform (which I don't think are any that support the alternative vtable format yet).